tests/test_main.py

```diff
@@ -949,3 +950,83 @@ def test_volumes_alias_override(self):
         out, _ = self.run_scuba(['doit'])
         out = out.splitlines()
         assert out == ["from the alias"]
+
+    def test_volumes_host_path_create(self):
+        '''Missing host paths should be created before starting Docker'''
+
+        userdir = Path('./user')
+        testfile = userdir / 'test.txt'
+
+        with open('.scuba.yml', 'w') as f:
+            f.write('''
+                image: {image}
+                volumes:
+                  /userdir: {user_dir}
+                '''.format(image = DOCKER_IMAGE,
+                           user_dir = userdir.absolute(),
+                           ))
+
+        self.run_scuba(['touch', '/userdir/test.txt'])
+
+        assert os.path.exists(testfile.absolute()), "Test file was not created"
+
+        info = os.stat(userdir)
```

Comment:
Take advantage of the fat that `userdir` is a `Path`:
```suggestion
        info = userdir.stat()
```